maintenance/recipes/turn_off.rb

```diff
@@ -0,0 +1,3 @@
+bash 'turn maintenance off' do
+  code 'sudo -S -u deploy rm /srv/www/jiffyshirts/current/public/offline'
```

Comment:
I believe Chef agent is already sudoed so you don't need to `sudo -S -u deploy`.

Reply:
we tried without `deploy` user - it did not work

Reply:
Hm ok. Anyway - next comment is a more natural way to do that.